### PR TITLE
Update version of MeshToolkit used in tests

### PR DESCRIPTION
**Pull request by mmisol**
### Purpose

This updates to version 3.0.0.0 which uses latest VC++ redistributable.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

File: test/core/userdata/0.8/packages/MeshToolkit/extra/backup/ConvertMeshIntoLinePointsSurfaces.dyn.0.backup [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<Workspace Version="0.8.2.1957" X="26125.1497935841" Y="22672.9170313985" zoom="1.15636491448696" Name="Home" RunType="Manual" RunPeriod="1000" HasRunWithoutCrash="True">
```
Comment (QilongTang) [inline test/core/userdata/0.8/packages/MeshToolkit/extra/backup/ConvertMeshIntoLinePointsSurfaces.dyn.0.backup:1]: Is `backup` folder created by you?

Comment (mmisol) [reply test/core/userdata/0.8/packages/MeshToolkit/extra/backup/ConvertMeshIntoLinePointsSurfaces.dyn.0.backup:1]: My bad. That seems to be created automatically when I run the tests. I'll delete that from the PR

File: test/Libraries/WorkflowTests/PackageValidationTest.cs [hunk 2]
```diff
@@ -89,7 +89,7 @@ public void TestTextFromDynamoText()
         public void ConvertMeshIntoLinePointsSurfaces()
         {
             var testFilePath = Path.Combine(TestDirectory,
-    @"core\userdata\0.8\packages\MeshToolkit.0.8.2\extra\ConvertMeshIntoLinePointsSurfaces.dyn");
+    @"core\userdata\0.8\packages\MeshToolkit\extra\ConvertMeshIntoLinePointsSurfaces.dyn");
```
Comment (QilongTang) [inline test/Libraries/WorkflowTests/PackageValidationTest.cs:92]: @mmisol Did we also update the MeshToolKit version? If yes, then I prefer not putting it in `0.8` folder anymore but reflect the actual version here

Comment (mmisol) [reply test/Libraries/WorkflowTests/PackageValidationTest.cs:92]: Ok, I could do that. Should I do something to the other packages in that folder? We have:
- List.SortByKeyList version 0.0.2
- Dynamo Text version 0.1.0

Comment (mmisol) [reply test/Libraries/WorkflowTests/PackageValidationTest.cs:92]: Thinking about it, maybe `0.8` refers to the version of Dynamo? Would that explain why the `backup` folder got created?

Comment (QilongTang) [reply test/Libraries/WorkflowTests/PackageValidationTest.cs:92]: I would leave them there, I think we have another task for Dynamo Text package.

**Comment by mmisol** [comment]
Successful build here: https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/653/

Will hold on merging until you let me know @QilongTang

**Pull request merged**